packages/now-cli/test/dev/fixtures/output-is-source/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "build": "rm -Rf public && mkdir -p public && echo hello first > public/index.html"
```

Comment:
Are you missing quotes aroun `echo 'hello' first` or probably `echo 'hello first'`?

Reply:
`echo` prints every argument as one string, it doesn't have to be one

Reply:
But I added them, to make it easier to read